src/tests/catwalk_tests/test_individual_importance_uniform.py

```diff
@@ -5,12 +5,14 @@
 from triage.component.catwalk.db import ensure_db
 from triage.component.catwalk.storage import InMemoryMatrixStore
 from triage.component.catwalk.individual_importance.uniform import uniform_distribution
+from triage.component.results_schema import Matrices
 
 from tests.results_tests.factories import (
     ModelFactory,
     FeatureImportanceFactory,
     init_engine,
     session,
+    MatrixFactory
```

Comment:
Same in this file. It should require no changes with the way the ModelFactory invokes the MatrixFactory.